src/Apps/Sell/SellFlowContext.tsx

```diff
@@ -27,6 +27,11 @@ const BASIC_FLOW_STATES: ConsignmentSubmissionStateAggregation[] = [
   "DRAFT",
   "SUBMITTED",
 ]
+// If a submission is in one of these states, additional information is required.
+export const PRE_SUBMITTED_STEPS: ConsignmentSubmissionStateAggregation[] = [
```

Comment:
What do you think about the naming? I would be happy to hear any suggestions for a better or more precise name for these two steps.